src/api/Alert/types.ts

```diff
@@ -162,13 +162,20 @@ export enum GetCasesSortBy {
   FOUR = '4'
 }
 
+export enum AssigneeFilteringOptions {
+  ALL,
+  ME,
+  UNASSIGNED
+}
```

Comment:
@EricMallard do you still need this enum ?

Reply:
nope!